src/config/scrivitoSites.ts

```diff
@@ -85,7 +121,7 @@ function baseUrlsFor(site: Obj) {
 }
 
 export function isNoSitePresent(): boolean {
-  return !appWebsites()?.length
+  return !getLanguageVersions()?.length
 }
 
 export async function ensureSiteIsPresent() {
```

Comment:
As discussed in a voice call: How about extracting `ensureSiteIsPresent` to a dedicated module, that should only know about finding a good language version for a detected content ID?